### ISSUE TITLE
Not working with symfony serve webserver

### ISSUE THREAD

**Issue by beberlei** [shopware/shopware#115]
When I use `symfony` tool as a webserver with `serve` I get an error after redirect to /recover/install in the console:

```
PHP Fatal error:  Uncaught Symfony\Component\Dotenv\Exception\PathException: Unable to read the "/var/www/shopware/public/../.env" environment file. in /var/www/shopware/vendor/symfony/dotenv/Dotenv.php:484 
```
Then if i touch `.env`, i get another one that Doctrine PDO MySQL is wrongly configured.

```
The options 'driver' or 'driverClass' are mandatory if no PDO instance is given to DriverManager::getConnection().
```
Please just close if symfony local webserver is just not supported.

**Comment by janbuecker** [comment]
Thanks for the report @beberlei. We've never used the symfony webserver internally so it could be broken. But I think it might be a good idea to support it. 

We'd love to receive a pull request which fixes this issue. Otherwise we'll take a look into it within the next weeks. :)

**Comment by dnl-jst** [comment]
Would be a really good idea to support it. It's much faster than docker or vagrant on macOS because it runs directly on the machine without any virtualization.

**Comment by dnl-jst** [comment]
Got it working with symfony binary. Had the same issue as @beberlei, but my mistake was to try to get an already setup'ed application to run with symfony cli.

What I did:

`git clone git@github.com:shopware/development.git`
`cd development`
`git clone git@github.com:shopware/platform.git`
`symfony php bin/setup`

As `URL to your /public folder` I used `https://localhost:8000`

`symfony server:start`

The above setup assumes that there is no other symfony server running and we get the first free port 8000.
